configs/datasets/pointnav/gibson_v2.yaml

```diff
@@ -0,0 +1,11 @@
+ENVIRONMENT:
+  MAX_EPISODE_STEPS: 500
+SIMULATOR:
+  AGENT_0:
+    HEIGHT: 0.88
+    RADIUS: 0.18
+
+DATASET:
+  TYPE: PointNav-v1
```

Comment:
Shouldn't we reserve v2 for functional changes to the dataset class? Maybe it more like PointNav 1.1?

Reply:
Great question, class itself wasn't changed. That's why v1 should be fine.